pyblish_bumpybox/plugins/collect_existing_files.py

```diff
@@ -34,103 +31,120 @@ def version_get(self, string, prefix):
             return None
         return matches[-1:][0][1], re.search("\d+", matches[-1:][0]).group()
 
-    def scan_versions(self, instance_collection, version):
+    def get_version_collections(self, collection, version):
+        """Return all collections of previous collection versions."""
 
-        # Getting collections of all previous versions and current version
         collections = []
         for count in range(1, int(version) + 1):
-
-            # Generate collection
             version_string = "v" + str(count).zfill(len(version))
-            head = instance_collection.head.replace(
+            head = collection.head.replace(
                 "v" + version, version_string
             )
-            collection = clique.Collection(
+
+            version_collection = clique.Collection(
                 head=head.replace("\\", "/"),
-                padding=instance_collection.padding,
-                tail=instance_collection.tail
+                padding=collection.padding,
+                tail=collection.tail
             )
-            collection.version = count
+            version_collection.name = collection.name
```

Comment:
No, the ```clique.Collection``` method does not work like that.

Assigning the variables to the collection object is a work around for how the code is structured.